README.md

```diff
@@ -342,6 +342,11 @@ select * from users
 select * from users where user_id = $1
 ```
 
+Multiple fragments can be joined together with `sql.join` to create more complex dynamic queries:
+```js
+const columns = ['id', 'name', 'age'].map(name => sql(name))
+sql`select ${sql.join(', ', columns)} from users where
```

Comment:
I think this example line should be:

    sql`select ${sql.join(sql`, `, columns)} from users where

Doesn't the separator need to be a sql clause?